src/devices/vdml_aivision.c

```diff
@@ -0,0 +1,161 @@
+/**
+ * \file devices/vdml_aivision.c
+ *
+ * Contains functions for interacting with the V5 Vision Sensor.
+ *
+ * \copyright Copyright (c) 2017-2024, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include "v5_api.h"
+#include "vdml/registry.h"
+#include "vdml/vdml.h"
+#include "pros/aivision.h"
+
+
+int32_t aivision_reset(uint8_t port) {
+    claim_port_i(port - 1, E_DEVICE_AIVISION);
+    vexDeviceAiVisionReset(device->device_info);
+    return_port(port - 1, PROS_SUCCESS);
+}
+
+int32_t aivision_set_enabled_detection_types(uint8_t port, uint8_t bits, uint8_t bitmask) {
+    claim_port_i(port - 1, E_DEVICE_AIVISION);
+    vexDeviceAiVisionEnableSet(device->device_info, bits, bitmask);
+    return_port(port - 1, PROS_SUCCESS);
+}
+
+int32_t aivision_get_enabled_detection_types(uint8_t port) {
+    claim_port_i(port - 1, E_DEVICE_AIVISION);
+	  uint8_t enabled_detection_types = vexDeviceAiVisionEnableGet(device->device_info);
+	  return_port(port - 1, enabled_detection_types);
```

Comment:
```suggestion
    claim_port_i(port - 1, E_DEVICE_AIVISION);
    uint8_t enabled_detection_types = vexDeviceAiVisionEnableGet(device->device_info);
    return_port(port - 1, enabled_detection_types);
```
Inconsistent spacing